importers/data_sources/importer.py

```diff
@@ -0,0 +1,153 @@
+"""
+Loads data sources into RE.
+This is a simple importer, as the data source load data is quite simple, 
+and also in it's ultimate form, requiring no transformation.
+All that is required is to:
+- read in the source  data json
+- uses the source_data schema to validate
+- save as separate files, one  per node
+- load and import those nodes 
+- using the same source_data schema to validate
+"""
+import argparse
+import json
+import os
+import traceback
+import requests
+import importers.utils.config as config
+from relation_engine_server.utils.json_validation import (
+    get_schema_validator,
+)
+
+
+class Importer(object):
+    def __init__(self):
+        pass
+
+    def get_config_or_fail(self, key):
+        if not hasattr(self, "_config"):
+            self._configure()
+
+        if key not in self._config:
+            raise KeyError(f'No such config key: "{key}"')
+
+        return self._config[key]
+
+    def get_config(self, key, default_value):
+        if not hasattr(self, "_config"):
+            self._configure()
+
+        if key not in self._config:
+            return default_value
+
+        return self._config[key]
+
+    def _configure(self):
+        self._config = config.load_from_env(extra_optional=["ROOT_DATA_PATH"])
+        return self._config
+
+    def get_dataset_schema_dir(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        # TODO:  factor out "data_sources"
+        return os.path.join(
+            dir_path, "../", "../", "spec", "datasets", "data_sources"
+        )
+
+    def get_relative_dir(self, path):
```

Comment:
Pure-ish functions like this that make no use of `self` should either be class methods, or could be moved out of the class as a regular functions, so we can test or reuse them without needing an instance.

Reply:
That (and another method get_dataset_schema_dir) have since been moved out of the class.